google-api-client-assembly/dependencies/google-api-client-appengine-dependencies.html

```diff
@@ -96,22 +96,22 @@ <h3>test<a name="test"></a></h3>
 <th>License</th></tr>
 <tr class="b">
 <td>com.google.appengine</td>
-<td><a class="externalLink" href="http://code.google.com/appengine/appengine-api-labs/">appengine-api-labs</a></td>
+<td><a class="externalLink" href="https://cloud.google.com/appengine/docs/standard/java/quickstart">appengine-api-labs</a></td>
 <td>1.7.7</td>
 <td>jar</td>
-<td><a class="externalLink" href="http://code.google.com/appengine/terms.html">Google App Engine Terms of Service</a></td></tr>
+<td><a class="externalLink" href="https://cloud.google.com/terms/">Google App Engine Terms of Service</a></td></tr>
 <tr class="a">
 <td>com.google.appengine</td>
-<td><a class="externalLink" href="http://code.google.com/appengine/appengine-api-stubs/">appengine-api-stubs</a></td>
+<td><a class="externalLink" href="https://mvnrepository.com/artifact/com.google.appengine/appengine-api-stubs">appengine-api-stubs</a></td>
```

Comment:
Let's link to search.maven.org instead of mvnrepository (IIRC, this is a third-party site).